src/pages/index.md

```diff
@@ -2,5 +2,19 @@
 title: About Fedi Devs
 id: about
 slug: /
-sidebar_position: 1
 ---
+
+# Fediverse Developer Network
+
+## What is it?
+
+- Build a community around those building for the Fediverse.
+- Encourage interoperability between projects using the existing standards.
+- Create a place where real-world examples and best practices around federating between projects can be documented.
```

Comment:
... where the developer community maintains a single, shared repository of insights, examples, and best practices, instead of everybody having scattered information all over the place. Or something like that.